tests/05-sim-long_jumps.py

```diff
@@ -30,21 +31,27 @@
 
 def test(args):
     f = SyscallFilter(KILL)
-    # syscalls referenced by number to make the test simpler
-    f.add_rule_exactly(ALLOW, 1)
+    f.add_rule(ALLOW, "brk")
     i = 0
     while i < 100:
-        f.add_rule_exactly(ALLOW, 1000,
-                           Arg(0, EQ, i),
-                           Arg(1, NE, 0),
-                           Arg(2, LT, sys.maxsize))
+        f.add_rule(ALLOW, "chdir",
+                   Arg(0, EQ, i),
+                   Arg(1, NE, 0),
+                   Arg(2, LT, sys.maxsize))
         i += 1
     i = 100
```

Comment:
Two more nitpicks and then I'll shut up :)

In the python test, you're restarting the iter at `100` here, but in the C test, you're restarting at 0.  I think `100` makes sense.

Reply:
> Two more nitpicks and then I'll shut up :)

Ha, no worries, I'd rather we get this right :)

> In the python test, you're restarting the iter at 100 here, but in the C test, you're restarting at 0. I think 100 makes sense.

FWIW, the reason for starting at 0 was to catch all of the arch/ABIs that start their syscall numbering at 0, e.g. x86_64.  We go up to 10,000 simply because of the arch/ABIs that have an offset. We limit the number of syscalls added to the filter simply so we don't bump into the seccomp filter size limit.

Regardless, (re)starting at 0 seems like the correct thing to do here, I'll fix that.

Reply:
Sounds good.  Thanks for the clarification.